sortinghat/cli/cmds/lock.py

```diff
@@ -75,22 +75,22 @@ def _lock_identity(conn, **kwargs):
 @click.argument('uuid')
 @click.pass_obj
 def rm(client, uuid):
-    """Remove a lock from a unique identity so it can be modified.
+    """Remove a lock from a individual so it can be modified.
 
-    This command removes a lock from the unique identity
+    This command removes a lock from the individual
     identified by <uuid>. Therefore, this and its related
     entities such as identities, enrollments or the profile
     can be modified.
 
-    UUID: identifier of the unique identity which will be unlocked
+    UUID: identifier of the individual which will be unlocked
     """
     with connect(client) as conn:
         _unlock_identity(conn, uuid=uuid)
         display('lock.tmpl', uuid=uuid, unlocked=True)
 
 
 def _unlock_identity(conn, **kwargs):
-    """Run a server operation to unlock a unique identity."""
+    """Run a server operation to unlock a individual."""
```

Comment:
Same as above (`an individual`). Moreover, I think this should be renamed too to `_unlock_individual`.